module/dal/control/src/main/java/org/openbase/bco/dal/control/layer/unit/scene/SceneControllerImpl.java

```diff
@@ -446,12 +457,25 @@ private void verifyAllStates() {
 
         private void verifyState(final ServiceProvider<? extends Message> unit, final Message serviceState) throws VerificationFailedException {
 
+            // skip verification on destroyed required action observer!
+            if(destroy) {
+                return;
+            }
+
             if (!responsibleAction.isValid()) {
-                throw new VerificationFailedException("Action not valid anymore!");
+                throw new VerificationFailedException("The activation of "+getLabel("?") + " is not valid anymore.");
```

Comment:
This line is not properly formatted. Maybe re-run automatic formatting on this file.

```suggestion
                throw new VerificationFailedException("The activation of " + getLabel("?") + " is not valid anymore.");
```